addons/hr_timesheet/report/timesheets_analysis_report.py

```diff
@@ -25,6 +25,7 @@ class TimesheetsAnalysisReport(models.Model):
     unit_amount = fields.Float("Time Spent", readonly=True)
     partner_id = fields.Many2one('res.partner', string="Partner", readonly=True)
     milestone_id = fields.Many2one('project.milestone', related='task_id.milestone_id')
+    message_partner_ids = fields.Many2many('res.partner', readonly=True)
```

Comment:
```suggestion
    message_partner_ids = fields.Many2many('res.partner', compute="_compute_message_partner_ids", search="_search_message_partner_ids", readonly=True)
    
    def _compute_message_partner_ids(self):
        # duplicate code in account.analytic.line
        ...
        
    def _search_message_partner_ids(self, operator, value):
        return self.env['account.analytic.line']._search_message_partner_ids(operator, value)
```

Reply:
Didn't know we could add regular compute and search in report models.
I've added their definition (which just calls the one from `account.analytic.line`, is there a specific reason why you've noted that for the compute the code should be duplicated instead of just calling the other compute?